pipeline-scripts/uperf.groovy

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env groovy
+
+def pipeline_id = env.BUILD_ID
+def node_label = NODE_LABEL.toString()
+def run_uperf = UPERF.toString().toUpperCase()
+def property_file_name = "uperf.properties"
+
+println "Current pipeline job build id is '${pipeline_id}'"
+
+// run uperf
+stage ('uperf') {
+		if (run_uperf == "TRUE") {
+		currentBuild.result = "SUCCESS"
+		node(node_label) {
+			// get properties file
+			if (fileExists(property_file_name)) {
+				println "Looks like the property file already exists, erasing it"
+				sh "rm ${property_file_name}"
+			}
+			// get properties file
+			sh "wget ${UPERF_PROPERTY_FILE} -O ${property_file_name}"
+			sh "cat ${property_file_name}"
+                        def uperf_properties = readProperties file: property_file_name
+                        def skip_tls = uperf_properties['SKIP_TLS_VERIFICATION']
+			def cluster_user = uperf_properties['CLUSTER_USER']
+			def cluster_password = uperf_properties['CLUSTER_PASSWORD']
+			def cluster_api_url = uperf_properties['CLUSTER_API_URL']
+			def es_server = uperf_properties['ES_SERVER']
+			def es_port = uperf_properties['ES_PORT']
+			def metadata_collection = uperf_properties['METADATA_COLLECTION']
+			
+			try {
+				uperf_build = build job: 'RIPSAW-UPERF',
+				parameters: [   [$class: 'LabelParameterValue', name: 'node', label: node_label ],
+                                                [$class: 'BooleanParameterValue', name: 'SKIP_TLS_VERIFICATION', value: Boolean.valueOf(skip_tls) ],
+						[$class: 'StringParameterValue', name: 'CLUSTER_USER', value: cluster_user ],
+						[$class: 'StringParameterValue', name: 'CLUSTER_PASSWORD', value: cluster_password ],
+						[$class: 'StringParameterValue', name: 'CLUSTER_API_URL', value: cluster_api_url ],
+						[$class: 'StringParameterValue', name: 'ES_SERVER', value: es_server ],
+						[$class: 'StringParameterValue', name: 'ES_PORT', value: es_port ],
+						[$class: 'BooleanParameterValue', name: 'METADATA_COLLECTION', value: Boolean.valueOf(metadata_collection) ],
+			} catch ( Exception e) {
+				echo "UPERF Job failed with the following error: "
+				echo "${e.getMessage()}"
+				mail(
+					to: 'nelluri@redhat.com',
```

Comment:
Add yourself and Joe's email id. @jtaleric should we create a mailing list for receiving the email like you suggested in the trigger PR?